### PR TITLE
Enh: Allow predefined tags, attributes and non-ASCII characters

### LINKED ISSUE #49: Allow Symbol Character Translations
Currently, the module is wonderful when it comes to translating into other languages, the only issue that I'm seeing so far is symbol characters such as `,`, which in other languages also have their own translation which is purified through this module and doesn't allow for a true and complete translation.

#### Example
- Japanese
`,`  > `、`
`""` > `「」`
`%` > `％`
`~` > `～`

These are just some of the symbol characters that are purified through the module which makes translating very difficult.

### PR DESCRIPTION
**Pull request by ArchBlood**
Alternative for #43 but also allows for non-ASCII characters to be used when translating in other languages.

resolves #49

### CODE REVIEW

File: models/parser/TranslationPurifier.php [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-    public static function configure( $config)
+    public static function configure($config)
     {
-        // https://stackoverflow.com/questions/4566301/htmlpurifier-with-an-html5-doctype
+        // Set HTMLPurifier configuration
         $config->set('HTML.Doctype', 'HTML 4.01 Transitional');
-        $def =  $config->getHTMLDefinition(true);
-        $def->addAttribute('a', 'href', new ParameterURIDef());
-        $def->addAttribute('img', 'src', new ParameterURIDef());
+        $config->set('Attr.EnableID', true);
```
Comment (ArchBlood) [inline models/parser/TranslationPurifier.php:16]: See the following Docs;
http://htmlpurifier.org/live/configdoc/plain.html#Attr.EnableID

File: models/parser/TranslationPurifier.php [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+
+        // Allow specific tags and attributes
+        $config->set('HTML.Allowed', 'p,b,i,u,s,a[href|target],img[src|alt],ul,ol,li,blockquote,code,pre,span,hr,br,strong');
```
Comment (ArchBlood) [inline models/parser/TranslationPurifier.php:19]: Also see the following Docs;
http://htmlpurifier.org/live/configdoc/plain.html#HTML.Allowed

File: models/parser/TranslationPurifier.php [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+        
+        // Allow non-ASCII characters
+        $config->set('Core.EscapeNonASCIICharacters', false);
```
Comment (ArchBlood) [inline models/parser/TranslationPurifier.php:22]: And finally see this part of the Docs;
http://htmlpurifier.org/live/configdoc/plain.html#Core.EscapeNonASCIICharacters

**Comment by luke-** [comment]
@ArchBlood Thanks, let's try :-)

**Pull request merged**